src/packages/@ncigdc/components/analysis/CreateAnalysis.js

```diff
@@ -99,7 +99,9 @@ const CreateAnalysis = ({
                 <h1 style={{ fontSize: '2rem' }}>{item.label}</h1>
                 <div style={{ marginBottom: 10 }}>{item.description}</div>
                 <Row spacing={5}>
-                  <Button onClick={() => setAnalysis(item)}>Select</Button>
+                  {(DISPLAY_GENE_EXPRESSION &&
```

Comment:
hide "Select" button for gene expression. added the feature flag to make this change easier to find in the future.

other changes in this file are from eslint.